litecov.cpp

```diff
@@ -401,9 +404,9 @@ bool LiteCov::HasNewCoverage() {
 
 void LiteCov::OnProcessExit() {
   TinyInst::OnProcessExit();
-  if (IsTargetAlive()) {
-    CollectCoverage();
-  }
+  CollectCoverage();
+
+  TinyInst::ClearSharedMemory();

```

Comment:
I think it would be cleaner to rework this as

```
void LiteCov::OnProcessExit() {
  CollectCoverage();	
  TinyInst::OnProcessExit();
}
```

(i.e. call CollectCoverage() before the parent OnProcessExit) and then clear shared memory in Debugger::OnProcessExit. This way, we maintain shared memory completely independent of the LiteCov module.